querygen/querygen_test.go

```diff
@@ -20,7 +20,8 @@ func TestGenBuildPipelineRunCreatedQuery(t *testing.T) {
 	assert.NotEqual(t, "", out)
 }
 
-func TestGenBuildPipelineRunStartedQuery(t *testing.T) {
-	out := GenBuildPipelineRunStartedQuery("some_index")
+func TestGenBuildPipelineRunCompletedQuery(t *testing.T) {
```

Comment:
As far as I can tell we still need a test for `GenBuildPipelineRunStartedQuery` because we still use that function

Reply:
Done

Reply:
@ShakedAviv50313 we're now missing `TestGenBuildPipelineRunCreatedQuery`. Please ensure we have 3 tests for build PipelineRuns and that they call their respective query generation funcs.

Reply:
Done, I rebased this branch with upstream/main